src/DotVVM.Framework/ResourceManagement/IPreloadResource.cs

```diff
@@ -0,0 +1,13 @@
+﻿using DotVVM.Framework.Hosting;
+
+namespace DotVVM.Framework.ResourceManagement
+{
+    /// <summary>
+    /// Represents a resource thas is preloaded. 
```

Comment:
Maybe a more descriptive documentation would be nice, there is a few things to note:

* prerendered means a `<link rel="preload" ...>` is rendered into header.
* prerendered means that it is also rendered in the specified destination
* it does not have to be prerendered (for example when the resource is added after head is rendered)